### PR TITLE
Add accessibility ids

### LINKED ISSUE #15187: Add accessibilty-id for options in community
# Bug Report

## Problem 
no accessibility-id for community options
**Blocks 702845 e2e**

image

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by ibrkhalil**
fixes #15187, #15185

status: wip

### CODE REVIEW

File: src/quo2/components/common/unread_grey_dot/view.cljs [hunk 1]
```diff
@@ -5,4 +5,5 @@
 (defn unread-grey-dot
   []
   [rn/view
-   {:style style/unread-grey-dot}])
+   {:style               style/unread-grey-dot
```
Comment (J-Son89) [inline src/quo2/components/common/unread_grey_dot/view.cljs:8]: just incase you're not aware of this, sometimes you have to add `:accessible true` for the accessibility label to work on `rn/view`'s

**Comment by flexsurfer** [comment]
hey @ibrkhalil still relevant ?

**Comment by ibrkhalil** [comment]
> hey @ibrkhalil still relevant ?

Let's wait for @churik to check if it's and then we can close if it's not

**Comment by churik** [comment]
@ibrkhalil 
as PR is marked as draft, I supposed it is not ready yet.

Still no accessibility-ids fro community options:
image

For unread notifications IDs are in place.
Thanks!

**Comment by ibrkhalil** [comment]
> @ibrkhalil as PR is marked as draft, I supposed it is not ready yet.
> 
> Still no accessibility-ids fro community options: image
> 
> For unread notifications IDs are in place. Thanks!

I asked a question on the issue [here](https://github.com/status-im/status-mobile/issues/15187#issuecomment-1445137129) and was waiting for your reply .. Maybe you missed it ?

**Comment by churik** [comment]
@ibrkhalil answered [here](https://github.com/status-im/status-mobile/issues/15187#issuecomment-1477849467), sorry for delay

**Comment by ibrkhalil** [comment]
Closing because it's going to be fixed in another PR
https://github.com/status-im/status-mobile/pull/15485
https://github.com/status-im/status-mobile/pull/15484

**Pull request closed without merging**